acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/web3j/privacy/EnclaveErrorAcceptanceTest.java

```diff
@@ -32,7 +32,7 @@
 import org.junit.Before;
 import org.junit.Test;
 
-public class PrivateTxEnclaveErrorAcceptanceTest extends PrivacyAcceptanceTestBase {
+public class EnclaveErrorAcceptanceTest extends PrivacyAcceptanceTestBase {
```

Comment:
Did this file get renamed?  It's unclear from the Github commit.

Reply:
Yes, the file was renamed. It should be `PrivateTransactionEnclaveErrorAcceptanceTest` but that is a mouthful so I have gone with `EnclaveErrorAcceptanceTest` which describes the test well